citestwheel.Dockerfile

```diff
@@ -19,14 +19,24 @@ ARG DEBIAN_FRONTEND=noninteractive
 ENV PYENV_ROOT="/pyenv"
 ENV PATH="/pyenv/bin:/pyenv/shims:$PATH"
 
-RUN apt-get update \
-        && apt-get upgrade -y \
-        && apt-get install -y --no-install-recommends \
-        wget curl git jq ssh \
-        make build-essential libssl-dev zlib1g-dev \
-        libbz2-dev libreadline-dev libsqlite3-dev wget \
-        curl llvm libncursesw5-dev xz-utils tk-dev unzip \
-        libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+RUN <<EOF
+set -e
+if [[ "${LINUX_VER}" == "ubuntu18.04" ]]; then
+  # update git > 2.17
+  apt-get install -y software-properties-common
```

Comment:
Will this pass before calling `apt-get update` for the first time?

Reply:
ahhh nope. good catch. I will fix